packages/catalog-model/src/entity/constants.ts

```diff
@@ -57,3 +57,50 @@ export const ANNOTATION_KUBERNETES_API_SERVER_CA =
  */
 export const ANNOTATION_KUBERNETES_AUTH_PROVIDER =
   'kubernetes.io/auth-provider';
+
+/**
+ * Annotation for specifying the link to the Kubernetes dashboard managing this cluster
+ *
+ * @public
+ */
+export const ANNOTATION_KUBERNETES_CA_FILE = 'kubernetes.io/ca-file';
+
+/**
+ * Annotation for specifying the oidc provider used to get id tokens for a Kubernetes cluster
+ *
+ * @public
+ */
+export const ANNOTATION_KUBERNETES_OIDC_TOKEN_PROVIDER =
+  'kubernetes.io/oidc-token-provider';
+
+/**
+ * Annotation for specifying boolean value for skip metric lookup.
+ *
+ * @public
+ */
+export const ANNOTATION_KUBERNETES_SKIP_METRICS_LOOKUP =
+  'kubernetes.io/skip-metrics-lookup';
+
+/**
+ * Annotation for specifying boolean value for skip tls verify.
+ *
+ * @public
+ */
+export const ANNOTATION_KUBERNETES_SKIP_TLS_VERIFY =
+  'kubernetes.io/skip-tls-verify';
+
+/**
+ * Annotation for specifying the dashboard url for a Kubernetes cluster.
+ *
+ * @public
+ */
+export const ANNOTATION_KUBERNETES_DASHBOARD_URL =
+  'kubernetes.io/dashboard-url';
+
+/**
+ * Annotation for specifying the dashboard app for a Kubernetes cluster.
+ *
+ * @public
+ */
+export const ANNOTATION_KUBERNETES_DASHBOARD_APP =
+  'kubernetes.io/dashboard-app';
```

Comment:
Hm. I have an ask.

The k8s annotations shouldn't be in here at all, arguably; only DEFAULT_NAMESPACE, ANNOTATION_VIEW_URL and ANNOTATION_EDIT_URL actually belong.

Could you place both the old and new k8s ones in kubernetes-common please? Then might I trouble you to add to the three old ones in here, ```* @deprecated Import this constant from `@backstage/plugin-kubernetes-common` instead```.

At that point you may want to break the changeset into two (or possibly even three, for model / common / backend), with dedicated wordings of what was done to the respective package

Reply:
Sure thing I will make the changes

Reply:
Done